### ISSUE TITLE
Add a macOS-like about window

### ISSUE THREAD

**Issue by Pixel556** [PuruVJ/macos-web#43]
This project is really awesome. Like, super awesome. Can you add a macOS monterey-like about window? And make it so that it opens when you click the apple logo, and click on "About this Mac". Also add system preferences and add the appearance option which gives access to UI settings, So add options on there to switch between macOS Catalina UI and Monterey UI.

**Comment by Froggo8311** [comment]
Just as a side note-
AFAIK, there is not already a method to have those buttons interactive yet. It is possible, but it is not yet implemented.

**Comment by Pixel556** [comment]
Thx @Froggo8311.

**Comment by PuruVJ** [comment]
Yeah, the core logic isn't there yet, but it is possible. A lot of work though, so I'll maybe keep this in backlog for when I get back to it. Thanks for the idea

**Comment by Froggo8311** [comment]
> Yeah, the core logic isn't there yet, but it is possible. A lot of work though, so I'll maybe keep this in backlog for when I get back to it. Thanks for the idea

I might be able to work on this, if I'm not too busy :)

**Comment by Froggo8311** [comment]
Side note- make the "Shut Down", "Log out", and "Reboot" buttons open the typical dialogue, then reload the page =>
```js
function reload() {
  location.reload();
}
```
